src/components/CoinListHeader/index.js

```diff
@@ -1,25 +1,129 @@
-import { Row } from "antd";
-import { ColumnHeader } from "./CoinHeader.styles";
+import React from "react";
+import { Row, Col } from "antd";
+import { FaCaretUp, FaCaretDown } from "react-icons/fa";
+import { RiHeartFill } from "react-icons/ri";
+import { ColumnHeader, FavIcon } from "./CoinHeader.styles";
 
-export const CoinListHeader = (props) => {
-  return (
-    <Row>
-      <ColumnHeader span={1}>
-        <div>Fav</div>
-      </ColumnHeader>
-      <ColumnHeader span={1}>
-        <div>Rank</div>
-      </ColumnHeader>
+export default class CoinListHeader extends React.Component {
+  state = {};
```

Comment:
I guess you plannig on having state here, if you dont i'd change this to a functional component and just pass in props to it